roles/kerberos5/templates/kadm5.acl.j2

```diff
@@ -0,0 +1 @@
+*/admin@DOMAIN.ALT      *
```

Comment:
Inconsistent use of  `{{ krb5_realm }}` and `DOMAIN.ALT` is a bit confusing. Shouldn't it be `{{ krb5_realm }}`, like, everywhere?

Reply:
Fixed.